pkg/uibackend/rest/dashboard_riskiest_assets_test.go

```diff
@@ -270,6 +294,28 @@ func createVMInfo(t *testing.T, instanceID, location string) *backendmodels.Asse
 	return &info
 }
 
+func createContainerInfo(t *testing.T) *backendmodels.AssetType {
+	t.Helper()
+	info := backendmodels.AssetType{}
+	err := info.FromContainerInfo(backendmodels.ContainerInfo{
+		Id: utils.PointerTo("d66da925d976b8caf60ea59c5ec75b1950f87d506144942cdbf10061052a8088"),
+		ContainerName: utils.PointerTo("hungry_mcclintock"),
+		Location: utils.PointerTo("gke-sambetts-dev-clu-sambetts-dev-nod-449204c7-gqx5"),
+	})
+	assert.NilError(t, err)
+	return &info
+}
+
+func createContainerImageInfo(t *testing.T) *backendmodels.AssetType {
+	t.Helper()
+	info := backendmodels.AssetType{}
+	err := info.FromContainerImageInfo(backendmodels.ContainerImageInfo{
+		Name: utils.PointerTo("ghcr.io/openclarity/vmclarity-orchestrator"),
```

Comment:
what about the image tag/digest? how it is represented?

Reply:
ContainerImageInfo has an ID field which will contain the digest. Right now we don't have a field that contains the tags. This is all UI related and I'm not sure we're 100% sure how we want to represent this stuff in the UI so this is mostly placeholder.

I need to sync with @paralta, @ramizpolic and @adamtagscherer and update the Container and ContainerInfo to include details in a generic way that works for both Docker, ContainerD and Kubernetes.